lua/lvim/config/_deprecated.lua

```diff
@@ -155,18 +155,40 @@ function M.post_load()
         end
         spec[old_key] = nil
 
-        message = message or string.format("Use `%s` instead.", alternative)
-        deprecate(
-          string.format("%s` in `lvim.plugins", old_key),
-          message .. " See https://github.com/folke/lazy.nvim#-migration-guide"
-        )
+        message = message or string.format("*%s* = [value]", alternative)
+        vim.schedule(function()
+          vim.notify_once(
+            string.format(
+              [[We switched from packer.nvim to lazy.nvim. `%s` in `lvim.plugins` is deprecated. Use `%s` instead.
+Example:
+`lvim.plugins = {... {... *%s* = [value] ...} ...}` ->
+`lvim.plugins = {... {... %s ...} ...}`
+See https://github.com/folke/lazy.nvim#-migration-guide"]],
+              old_key,
+              message,
+              old_key,
+              message
+            ),
+            vim.log.levels.WARN
+          )
+        end)
       end
     end
 
     if spec[1] and spec[1]:match "^http" then
       spec.url = spec[1]
       spec[1] = nil
-      deprecate("{ 'http...' }` in `lvim.plugins", "Use { url = 'http...' } instead.")
+
+      vim.schedule(function()
+        vim.notify_once(
+          [[We switched from packer.nvim to lazy.nvim. `"http..."` in `lvim.plugins` is deprecated. Use `url = "http..."` instead.
+Example:
+`lvim.plugins = {... { "http..." ...} ...}` ->
+`lvim.plugins = {... { url = "http..." ...} ...}`
+See https://github.com/folke/lazy.nvim#-migration-guide"]],
```

Comment:
I don't think these are that helpful without actual strings from the user :thinking:

Reply:
I'll change it to use the user's values

Reply:
now it looks like this ![image](https://github.com/LunarVim/LunarVim/assets/110467150/15922331-11fd-402b-a7db-df8a5a0813de)